### ISSUE TITLE
Stratcon - Map Presets

### ISSUE THREAD

**Issue by HammerGS** [MegaMek/mekhq#3669]
Nick is currently working on a project where each track will have a temperature range, and each hex will be of a particular terrain type (and at some point I'll even have a graphical representation of it). Now, AtB already has a bunch of map generator presets, but for this project we'll need more. 

https://docs.google.com/document/d/1KDim77PjlxQqBjjTvkfLqjbtH-UFDNZas9FM5RCtWe0/edit?usp=sharing

This is a document with some terrain categories, matched up with mapgen presets. Anyone who's interested should feel free to help out by generating some map presets (and sending them over to me somehow; make sure to document what you're working on in the google doc). Feel free to match up existing map generator presets with some of these terrain types.

Note that I haven't gotten around to putting actual jungle in as a map gen option yet, that will be "soon" (tm).

Please attach any presets as comments here, you'll likely need to zip them up.

**Comment by kuronekochomusuke** [comment]
Moon 
mostly baren with lunar theme.  no vegetation, only ice water,  minimal  buildings

- moon - low cliffs, some craters, some elevation,  some roughs, some sands, low ice
- moon-base - town, some fortifications, low cliffs, some craters, some elevation,  some roughs, some sands, low ice
[moon.zip](https://github.com/MegaMek/mekhq/files/11472346/moon.zip)

**Comment by SuperStucco** [comment]
Two things.  First, a big legacy AtB issue is the lack of high temperature conditions.  I have a submission somewhere on that.  I would REALLY like to see some variation into the high temperature range rather than just 'average' and 'don't lick your Mech' level cold.

Two, had a hankering for some technical writing the other week - don't judge - so maybe these would help:
[Biome Project.docx](https://github.com/MegaMek/mekhq/files/11472660/Biome.Project.docx)
[Biome selection.xlsx](https://github.com/MegaMek/mekhq/files/11472662/Biome.selection.xlsx)

I was about to start generating sets of map presets based on these and trying to shuffle the sets back and forth out of the standard folder, based on the system stats as described in the doc.

**Comment by Thom293** [comment]
Here are some presets for Plains, Steppe and Tundra.  If you want more for these biomes, let me know.  

I cant find my gmail account.  Could someone do a strikethrough on these on the Google Doc?  

[PlainsSteppeTundra.zip](https://github.com/MegaMek/mekhq/files/11510584/PlainsSteppeTundra.zip)

**Comment by Thom293** [comment]
Here are some for HIlls, Mountains and Sea. e.g.: Hills-RiverRoad, Hills-Residential, Mountain-RuinedTrenches, Mountain-TouristTown, Sea-CoastalFort, Sea-MangroveCliffs, Sea-Fjord.

[HillsMountainSea.zip](https://github.com/MegaMek/mekhq/files/11585765/HillsMountainSea.zip)

**Comment by NickAragua** [comment]
Thanks for the maps, @Thom293, integrated them into the existing set and updated the google doc. 
@kuronekochomusuke I'll need to spend a little time thinking about how to integrate moon biome (I do want to do it, but it'd be weird for it to show up in the middle of other terrain types)

**Comment by jauby** [comment]
I've made a few map presets I'd like to share:
**Arctic Desert, Cold Forest, Cold Hills, Cold Mountain, Frozen Sea.**

If you would like to see any changes to these presets don't hesitate to let me know and I'll do my best to accommodate

[ColdBiomes.zip](https://github.com/MegaMek/mekhq/files/11702291/ColdBiomes.zip)

**Comment by NickAragua** [comment]
Awesome, thanks.

My latest MegaMek PR here https://github.com/MegaMek/megamek/pull/4514 adds jungle and a variety of other terrain, so those who are adventurous can pull down that artifact and build map presets using the newly available terrain types.

**Comment by jauby** [comment]
Are you still looking for the hot presets to get finished up? I'm waiting on [MM PR #4538](https://github.com/MegaMek/megamek/pull/4538) to get merged and I can work on those last biomes in the doc (I'm relying on the map preview button pretty heavily to get the presets feeling just right).

**Comment by NickAragua** [comment]
Yeah, that'd be super helpful. Make sure to use the new jungle/desert options recently introduced!

**Comment by Thom293** [comment]
5 Bases, CF40-100, 2 Fortress CF60-150.  Different Biomes.

Note that I have not playtested any of these, so they might need tweaking.

[BaseFortress.zip](https://github.com/MegaMek/mekhq/files/11912435/BaseFortress.zip)

**Comment by jauby** [comment]
Some more presets - getting close to finishing that doc! I took the liberty of creating separate presets for jungle and desert for  the "hot" presets since they all said jungle/desert.

These presets are the result of some trial and error and heavy use of the map preview function in MM to get the settings looking really good on nearly every map generation, but of course if there's anything you'd like to see changed or any new biomes to add just say the word.

Here's what's included in this upload:
**BaseCold, BaseHotDesert, BaseHotJungle, CityCold, CityHotDesert, CityHotJungle, Desert, ForestHot, HillsHotDesert, HillsHotJungle, Jungle, MountainHotDesert, MountainHotJungle, SeaHotDesert, SeaHotJungle, TownCold, TownHotDesert, TownHotJungle**
[Mapgen Presets - JungleDesertCold.zip](https://github.com/MegaMek/mekhq/files/11931086/Mapgen.Presets.-.JungleDesertCold.zip)

**Comment by Thom293** [comment]
Ok, trying to compare what we have here and what is on the list, I think the only thing undone or not already existing in MM is cold sea.  So here is Cold Sea versions of Coastal Fort, Roads, Deltas, Fjords, Fjords Steep.  

[ColdSeaMaps.zip](https://github.com/MegaMek/mekhq/files/11934065/ColdSeaMaps.zip)

So I think that is all of them.

**Comment by NickAragua** [comment]
Ok, updated the google doc. The ones without "X" in front of them still could use a few more presets. Of note is that I added "Badlands" as a biome type as per discord discussion. Thanks for everyone's contributions.

**Comment by NickAragua** [comment]
I'll leave this open during playtesting so that folks can submit more presets.

**Comment by jauby** [comment]
Finished a few Biomes over the weekend. In this upload we have 8 arctic desert presets and 10 badlands presets.
A few presets worth explaining:
- BlastedLands: My attempt at a sort of destroyed or war torn landscape, lots of small craters, rubble, rough ground etc.
- ImpactSite: A large meteor impact site (basically just a large crater with a good amount of rubble)
- AbandonedTown: Less buildings than a "regular" town preset, less building CF, more rubble, some small craters.
- Outpost: Similar to a town, much less buildings, each building having much higher CF and some fortified ground
[ArcticDesert+Badlands.zip](https://github.com/MegaMek/mekhq/files/12065135/ArcticDesert%2BBadlands.zip)

**Comment by HammerGS** [comment]
Going to leave this open so players can find and add presets.
